src/app/component/user/components/profile/edit-profile/edit-profile.component.ts

```diff
@@ -20,7 +22,19 @@ export class EditProfileComponent implements OnInit {
       'My Credo is to make small steps that leads to huge impact. Let’s change the world together.',
   };
 
-  constructor() {}
+  constructor(private dialog: MatDialog) {}
 
   ngOnInit() {}
+
+  private openCancelPopup(): void {
```

Comment:
General: if you use methods in template set their access modifier to 'public', not 'private'. Change it in you files

Reply:
fixed